### PR TITLE
[etckeeper] add automatic email on commit

### PR DESCRIPTION
**Pull request by bleuchtang**
This PR add the possibility to send an email on every commit (daily commit or manual commit). It's disable by default so it didn't break anything. Work only on git but can be improve for other vcs in futur.

### CODE REVIEW

File: ansible/roles/etckeeper/defaults/main.yml [hunk 1]
```diff
@@ -285,6 +285,19 @@ etckeeper__avoid_commit_before_install: '{{ True
...
+#
+# Set this option to ``True`` to allow :program:`etckeeper` send email on every
+# commit.
+etckeeper__email_on_commit: False
+                                                                   # ]]]
+
+# .. envvar:: etckeeper__email_on_commit_email [[[
+#
+# Email address for :program:`etckeeper` to use in email send after commit.
+etckeeper__email_on_commit_email: '{{etckeeper__vcs_email}}'
```
Comment (ypid) [inline ansible/roles/etckeeper/defaults/main.yml]:
```yaml
etckeeper__email_on_commit_email: '{{ etckeeper__vcs_email }}'
```

to match the coding style.

**Review by ypid — DISMISSED** [review]
Otherwise it looks good.

File: ansible/roles/etckeeper/defaults/main.yml (hunk 1 above)
Comment (bleuchtang) [reply ansible/roles/etckeeper/defaults/main.yml]: Fix !

**Review by ypid — APPROVED** [review]
LGTM

**Review by drybjed — APPROVED** [review]
Nice implementation, thanks. I'll backport this to the stable releases as well.

**Pull request merged**